wintermotes/week5/ass6/app/components/fields.js

```diff
@@ -0,0 +1,220 @@
+var React = require('react');
```

Comment:
I see what your're getting at here, but this is a bit too mush splitting up.
The fields only differ with a text and callback.